apps/backend-sls/GraphQL/index.ts

```diff
@@ -1,9 +1,33 @@
+import { PrismaClient } from '@klicker-uzh/prisma'
 import serverlessExpress from '@vendia/serverless-express'
+import Redis from 'ioredis'
 
-import app from './app'
+import prepareApp from './app'
+
+const prisma = new PrismaClient()
```

Comment:
Prisma and Redis connections are initiated outside of the main handler that we return -> might be reused across serverless function instances, if we are lucky